ui_framework/components/local_nav/_local_search.scss

```diff
@@ -28,6 +28,10 @@
   }
 }
 
+.kuiLocalLuceneSearchInput:focus {
+  padding-right: 13em;
+}
```

Comment:
Will other plugins be able to use this class if we put it in `src/core_plugins/kibana/public/kibana.less`? If yes, then I think that's fine. If no, it would be nice to keep it somewhere reusable. 

By name it as a modifier, do you mean something like `kuiLocalSearchInput--lucene`?